### PR TITLE
Fix 1714: overrides for settings view

### PR DESCRIPTION
**Pull request by quangDecember**
## Summary of Changes

This pull request fixes issue #1714 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by quangDecember** [comment]
Hi I made a fix for #1714 
the problem was Settings page status bar seems to represent the OS, not the current app appearance, so I overrides it again

File: Client/Frontend/Settings/SettingsNavigationController.swift [hunk 1]
```diff
@@ -16,6 +16,13 @@ class SettingsNavigationController: UINavigationController {
     }
 
     override var preferredStatusBarStyle: UIStatusBarStyle {
+        if #available(iOS 13, *) {
+            if self.view.appearanceOverrideUserInterfaceStyle == .light || self.overrideUserInterfaceStyle == .light {
+                return .darkContent
+            } else {
```
Comment (Brandon-T) [inline Client/Frontend/Settings/SettingsNavigationController.swift]: This is so minor imo, but it's worth mentioning just for future reference.

```swift
if condition {
    return a
} else {
    return b
}

//can also be written as (without the else):

if condition {
    return a
}
return b
```

Anyway, thanks for the fix! :)
Really appreciated.

**Review by Brandon-T — APPROVED** [review]

**Review by iccub — CHANGES_REQUESTED** [review]
Please address the change that Brandon requested and we can merge this PR :)
Looks good

**Review by iccub — APPROVED** [review]
Thank you sir!

**Pull request merged**